tests/component/test_python_component.py

```diff
@@ -0,0 +1,32 @@
+import dask.dataframe as dd
+import pandas as pd
+import pyarrow as pa
+from fondant.component import DaskLoadComponent, python_component
+from fondant.pipeline import Pipeline
+
+
+def test_python_component():
+    pipeline = Pipeline(name="dummy-pipeline", base_path="./data")
+
+    @python_component(
+        base_image="python:3.8-slim-buster",
+        extra_requires=["pandas", "dask"],
+    )
+    class CreateData(DaskLoadComponent):
+        def __init__(self, *_, **__):
```

Comment:
This is needed to override the init of the `BaseComponent` but ideally it is not required in the future.